api_documentation/Core.postman_collection.json

```diff
@@ -1795,46 +1795,6 @@
 							},
 							"response": []
 						},
-						{
-							"name": "Update Type Relationship Pair",
```

Comment:
This deletion seems unrelated, did you mean to remove this API from the docs?

Reply:
I believe this was a duplicate. I still have one for the route showing up.